src/properties/font.rs

```diff
@@ -908,6 +912,42 @@ impl<'i> PropertyHandler<'i> for FontHandler<'i> {
   }
 }
 
+/// [`system-ui`](https://www.w3.org/TR/css-fonts-4/#system-ui-def) is a special generic font family
+/// It is platform dependent but if not supported by the target will simply be ignored
+/// This list is an attempt at providing that support
+#[inline]
+fn compatible_font_family(family: Option<Vec<FontFamily>>, is_supported: bool) -> Option<Vec<FontFamily>> {
```

Comment:
You could either return early here if `is_supported` is true, or not even call this function in that case. Either way it would avoid some extra checks.

Reply:
I chose the early return